res/qml/Theme/Theme.qml

```diff
@@ -4,17 +4,25 @@ pragma Singleton
 QtObject {
     property color accent: "#2D4EA1"
     property color accentColor: "#3a60be"
-    property color backgroundColor: "#2E2E2E"
+    property color backgroundColor: "#1e1e1e"
```

Comment:
We should think about a way to support selectable color schemes for QML -- light and dark mode at bare minimum.  It's fine to only have one mode to start with, but let's mark sure we can add light mode later.